spec/features/service_drafts/preview/empty/service_information_preview_spec.rb

```diff
@@ -23,12 +23,13 @@
   end
 
   it 'displays the correct progress indicators for required fields' do
+    anchors = ['name-label','long-name-label','type-label','version-label','description-label']
```

Comment:
In general, I don't have issues with running the test through a list, though it can create some issues. I think that you should put this into a `let` block (https://rspec.rubystyle.guide/#let-blocks), and you can probably use `%w[]` since the anchors don't have spaces (I'm surprised rubocop isn't yelling at you about it). You should make these changes to all the places this is being done.

I thought about declaring this in the helper files so you don't need to repeat, but haven't found anything to support doing it that way.